packages/compartment-mapper/src/bundle.js

```diff
@@ -332,6 +332,9 @@ ${importsCellSetter(__liveExportMap__, index)}\
     onceVar: {
 ${importsCellSetter(__fixedExportMap__, index)}\
     },
+    metaVar: {
+      url: '${packageLocation}'\
+    },
```

Comment:
What do we want to do about bundler?

Reply:
To support `import.meta` in the bundler, we’d inline the logic we’re introducing to `module-instance.js` in the generated code. `Object.assign($h_ort_meta, descriptor.meta)`. We’d need to capture the `meta` property and JSON serialize it into the bundle at that location.

For now, though, the bundler is very incomplete and can remain that way until we have a motivating use case, like replacing Browserify at MetaMask or replacing Rollup in Agoric SDK bootstrap. To get there, it needs a lot of work. For now, it’s sufficient for generating the SES bundles only. For example, it only works within a single package. It doesn’t support CommonJS (though, it probably could pretty easily).

Reply:
this is now potentially the only remaining todo unless we want to progress on changing the record envelope format

Reply:
So should we agree on how to proceed for bundle.js or should I remove this change and add a new issue?

Reply:
Agreed to not do anything and leave import.meta as an empty object.